internal/daysteps/daysteps.go

```diff
@@ -1,20 +1,60 @@
 package daysteps
 
+// Пакет содержит две функции:
+// одну экспортируемую
+// и одну неэкспортируемую — вспомогательную.
+
 import (
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/spentcalories"
 )
 
 const (
-	// Длина одного шага в метрах
-	stepLength = 0.65
-	// Количество метров в одном километре
-	mInKm = 1000
+	stepLength = 0.65	// Длина одного шага в метрах
+	mInKm = 1000// Количество метров в одном километре
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(data, ",") 
+	if len(parts) != 2 {
+		return 0, 0, errors.New("bad data format")
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil || steps <=0 {
+		return	0, 0 ,errors.New("bad data in steps")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```